korge/src/commonMain/kotlin/com/soywiz/korge/render/BatchBuilder2D.kt

```diff
@@ -870,5 +870,5 @@ class BatchBuilder2D constructor(
 }
 
 //internal const val BB_MAX_TEXTURES = 6
-internal const val BB_MAX_TEXTURES = 4
+internal const val BB_MAX_TEXTURES = 1
```

Comment:
We should probably configure this dynamically. This would prevent other targets from using multiple textures

Reply:
Oh dang it, this was supposed to be reverted for this pr, sorry!

Reply:
No problem. No need to revert anymore: https://github.com/korlibs/korge-next/blob/8e16ee712ceea65866e18aa0597f2fe5a94de358/korge/src/commonMain/kotlin/com/soywiz/korge/render/BatchBuilder2D.kt#L876-L879